tests/p_kernel/01_kernel_centos_keyring.sh

```diff
@@ -27,12 +27,19 @@ if [ "$centos_ver" -ge 7 ] ; then
   fi
   for id in kpatch "Driver update" kernel
   do
-    t_Log "Verifying x.509 CentOS ${id}"
-    keyctl list %:$ring | grep -i "CentOS \(Linux \)\?${id} signing key" > /dev/null 2>&1
+    t_Log "Verifying x.509 $os_name ${id}"
+    if [[ $is_almalinux == "yes" ]]; then
+      key="AlmaLinux ${id} signing key"
+    else
+      key="CentOS \(Linux \)\?${id} signing key"
```

Comment:
Same thing here, we might want to factor the OS-related vars into their own groups in 0_lib

Reply:
Same as in previous